esmvalcore/config/_dask.py

```diff
@@ -0,0 +1,65 @@
+"""Configuration for Dask distributed."""
+import contextlib
+import importlib
+import logging
+from pathlib import Path
+
+import yaml
+from distributed import Client
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILE = Path.home() / '.esmvaltool' / 'dask.yml'
```

Comment:
could you add support for central installation where this file would sit elsewhere and would have a certain configuration, prob site-specific? Not here, but keep it in mind (issue open etc), so we can decide how to do it